handler/oauth2/strategy_jwt_session.go

```diff
@@ -103,3 +103,14 @@ func (s *JWTSession) Clone() fosite.Session {
 
 	return deepcopy.Copy(s).(fosite.Session)
 }
+
+// GetExtraClaims implements ExtraClaimsSession for JWTSession.
+// The returned value is a copy of JWTSession claims.
+func (s *JWTSession) GetExtraClaims() map[string]interface{} {
+	if s == nil {
+		return nil
+	}
+
+	// We make a clone so that WithScopeField does not change the original value.
+	return s.Clone().(*JWTSession).GetJWTClaims().WithScopeField(jwt.JWTScopeFieldString).ToMapClaims()
```

Comment:
Why is the scope field hardcoded here?

Reply:
Because in fact the only standard approach for scope field is to do `JWTScopeFieldString` and not `JWTScopeFieldList`. The only reason why `JWTScopeFieldList` exists is for backwards compatibility. Because we are adding new output here, I do not think we have to care about backwards compatibility. Thus I just picked the standard behavior.